### DISCUSSION TITLE
Using a tauri app as a C++ library ?

### DISCUSSION THREAD

**Discussion by jacquesmignault** [tauri-apps/tauri#7414] [Q&A]
Hi !
I'm a noobie with Rust.
I saw this setting in Cargo.toml :

#[lib]
#crate-type = ["cdylib"]

Does this mean I can generate a dylib for another C++ app, made with say JUCE, to use ?
Thanks

**Comment by simonhyll** [comment]
https://github.com/tauri-apps/tauri/tree/dev/examples/tauri-dynamic-lib

**Reply by mildred** [reply]
It seems in the dev branch this was removed in commit 8de308d1bf6a855d7a26af58bd0e744938ba47d8 (#8723). Is this still possible for the next version of Tauri?

**Reply by simonhyll** [reply]
I have a vague memory of the reason it being removed was because it was misleading what you actually accomplished with it. There's nothing that I know of that prevents you from using the setup it had, it just did more harm than good and introduced maintenance overhead. But I could be wrong.

@amrbashir why was it removed?
